### PR TITLE
feat: Add analytic to most of the `policy` subcommands

### PR DESCRIPTION
**Pull request by candiduslynx**
#### Summary

Add analytics support to `policy` ssubcommands.
---

Use the following steps to ensure your PR is ready to be reviewed

- [x] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [x] Run `go fmt` to format your code 🖊
- [x] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [x] Update or add tests 🧪
- [x] Ensure the status checks below are successful ✅

### CODE REVIEW

**Review by roneli — APPROVED** [review]
LGTM, had a few comments overall.

**Comment by roneli** [comment]
I think the go mod tidy you didn't use -compat=1.17 which resulted in change in go.sum maybe this caused linting failure not 100%.

File: cmd/policy/validate.go [hunk 1]
```diff
@@ -13,24 +13,21 @@ const (
...
-			diags := c.ValidatePolicy(cmd.Context(), args[0])
-			errors.CaptureDiagnostics(diags, map[string]string{"command": "policy_validate"})
-			if diags.HasErrors() {
-				return fmt.Errorf("policy validate has one or more errors, check logs")
-			}
-			return nil
-		},
-	}
-	return cmd
+var validate = &cobra.Command{
```
[the remark this answers is not in the document]
Comment (yevgenypats) [reply cmd/policy/validate.go:16]: Yes, please revert back as cobra has a lot of issues when used in the simple/global way especially when working with viper.

[the remark this answers is not in the document]
Comment (yevgenypats) [reply cmd/policy/validate.go:16]: See pattern from kubectl but many other popular projcets take similar approach

File: cmd/policy/describe.go [hunk 2]
```diff
@@ -23,8 +23,8 @@ cloudquery policy describe github.com/COMMUNITY_GITHUB_ORG/aws
 `
 )
 
-func newCmdPolicyDescribe() *cobra.Command {
-	describePolicyCmd := &cobra.Command{
+func describe() *cobra.Command {
```
Comment (yevgenypats) [inline cmd/policy/describe.go:26]: why the change to the naming convention?

File: cmd/policy/validate.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (candiduslynx) [reply cmd/policy/validate.go]: reverted the code

File: cmd/policy/describe.go (hunk 2 above)
Comment (candiduslynx) [reply cmd/policy/describe.go]: reverted

**Comment by yevgenypats** [comment]
We can hold off with that for now as we are changing the architecture for policies anyway.

**Comment by yevgenypats** [comment]
Closing as we move policies to pure SQL
